docs/_components/modals-basic.html

```diff
@@ -0,0 +1,25 @@
+---
+title: "Basic"
+description: |
```

Comment:
@jmatembu 

Thank you for the feedback.

First, Thanks for pointing it out. I'll remove it from all my files. I copied/pasted the code from the [detailed instruction](https://github.com/BovAcademy-opensource/evolution-ui/blob/development/.github/CONTRIBUTING.md#add-a-component-to-the-showcase-website). You may want to edit it. 

Second, the fullscreen modal overlay should be in opaque light blue color. I think there may be something wrong after adding to the showcase page. In the original code, I only have one overlay for all modals. The overlay changes by toggle different classes. So I only added one overlay code to the basic modal html file. Should I add it to all modal html files?

Regarding the 2nd and 3rd points, how can I see the showcase page locally?

Thank you.